packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -127,7 +137,7 @@ contract EpochManager is
     _currentEpoch.startTimestamp = block.timestamp;
 
     elected = firstElected;
-    epochManagerInitializer = address(0);
+    epochManagerEnabler = address(0);
```

Comment:
no need to do this

Reply:
this was an ask from mariano
https://github.com/celo-org/celo-monorepo/pull/11187/files/9681185ab651c188009d5de9092b3ffbc2846f7c#diff-77cd3b411d09b04b34e1a9315ad80d540313578f9711c057a97d233bba26b4d4

Reply:
addressed in https://github.com/celo-org/celo-monorepo/pull/11207